static/templates/settings/add_new_custom_profile_field_form.hbs

```diff
@@ -36,6 +36,13 @@
             <label for="custom_field_url_pattern" >{{t "URL pattern" }}</label>
             <input type="url" id="custom_field_url_pattern" name="url_pattern" autocomplete="off" maxlength="1024" placeholder="https://example.com/path/%(username)s"/>
         </div>
+        <div class="input-group">
+            <label class="checkbox profile_field_display_label" for="profile_field_editable_by_user">
+                <input type="checkbox" id="profile_field_editable_by_user" name="editable_by_user" checked="checked"/>
+                <span></span>
+                {{t 'Users can edit this field' }}
```

Comment:
Can we use the existing `settings_checkbox` template here?

Reply:
Yes, now I have used `settings_checkbox`.